deployment/src/main/java/com/datastax/oss/quarkus/deployment/CassandraClientProcessor.java

```diff
@@ -116,64 +108,24 @@ BeanContainerListenerBuildItem build(
       BuildProducer<GeneratedBeanBuildItem> generatedBean) {
 
     feature.produce(new FeatureBuildItem(CASSANDRA_CLIENT));
-
-    String cassandraClientProducerClassName = getCassandraClientProducerClassName();
-    createCassandraClientProducerBean(generatedBean, cassandraClientProducerClassName);
+    String name = CassandraClientProducer.class.getName();
+    InputStream stream =
+        CassandraClientProcessor.class
+            .getClassLoader()
+            .getResourceAsStream(name.replace('.', '/') + ".class");
+
+    try {
+
+      generatedBean.produce(
+          new GeneratedBeanBuildItem(name, IOUtils.readFully(stream, stream.available())));
```

Comment:
This (and the InputStream setup above) represents a brain-dead simple way to try and accomplish the goals here.  I'm _sure_ there's a more elegant way to do this in Quarkus proper... I'm just experimenting at the moment